lib/config/validation.ts

```diff
@@ -578,18 +578,6 @@ export async function validateConfig(
                 }
               }
             }
-            if (key === 'fileMatch') {
```

Comment:
no longer needded? 🤔

Reply:
No, there is regexOrGlob option error handling present which handles the validation for all options with `patternMatch=true`